templates/PULL_REQUEST_TEMPLATE

```diff
@@ -0,0 +1,15 @@
+## Description
+
+<!--- Describe your changes in detail -->
+
+Fixes: <!--- add github link -->
+
+## How Has This Been Tested?
+
+<!--- Please describe how you tested your changes. -->
+<!--- Include details of your testing environment, and the tests you ran to -->
+<!--- see how your change affects other areas of the code, etc. -->
+
+## Screenshots (if appropriate):
```

Comment:
I find that this is very visualization tools specific, but would not be appropriate in most cases.  As this is for the template, it is okay, but I might change to (if appropriate to something more descriptive so someone copying knows that they can remove it)

Reply:
I'll add a comment to say that the section can be deleted if not appropriate.